### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by donia98**
Add validation for the first step inputs, email input in the second step and phone number input.

### CODE REVIEW

File: client/src/Components/Pages/Questionnaire/Wizard.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+
 export const Wizard = ({ step: currentIndex, ...props }) => {
   const steps = React.Children.toArray(props.children);
   const prevStep = currentIndex !== 0 && steps[currentIndex - 1].props;
   const nextStep = currentIndex !== steps.length - 1 && steps[currentIndex + 1].props;
 
+
   const SubmitQhandler = () => {
-     window.location=('/dash');
+    props.hanleUpdate();
```
Comment (maxgerber) [inline client/src/Components/Pages/Questionnaire/Wizard.js]: should rename this "handleUpdate" no?

File: client/src/Components/Pages/Questionnaire/index.js [hunk 2]
```diff
@@ -58,8 +63,31 @@ hanleUpdate() {
...
+
+  if (!parsedStorage.grade || parsedStorage.grade.trim() === '') {
+    alertify.dialog('alert').set({ transition: 'fade', message: 'Please enter your grade on the first step' }).setHeader('<h3>No grade !</h3>').show();
+  } else if (!parsedStorage.city || parsedStorage.city.trim() === '') {
+    alertify.dialog('alert').set({ transition: 'fade', message: 'Please enter your city on the first step' }).setHeader('<h3>No city !</h3>').show();
+  } else if (!parsedStorage.schoolEmail || parsedStorage.schoolEmail.trim() === '' || !validateEmail(parsedStorage.schoolEmail)) {
+    alertify.dialog('alert').set({ transition: 'fade', message: 'Please enter a valid school E-mail on the first step' }).setHeader('<h3>No school email !</h3>').show();
+  } else if (!parsedStorage.phoneNum) {
+    alertify.dialog('alert').set({ transition: 'fade', message: 'Please enter your phone number on the first step' }).setHeader('<h3>No phone num !</h3>').show();
+  // } else if (!(parsedStorage.emergencyEmail.trim() === '')) {
```
Comment (hshahwan) [inline client/src/Components/Pages/Questionnaire/index.js]: if you don't need it just delete it

File: client/src/Components/Pages/Questionnaire/index.js [hunk 3]
```diff
@@ -1,5 +1,10 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+<<<<<<< HEAD
+import alertify from 'alertifyjs';
+
+=======
```
Comment (hshahwan) [inline client/src/Components/Pages/Questionnaire/index.js]: I think you have conflict

**Comment by hshahwan** [comment]
@donia98 solve the conflict

File: client/src/Components/Pages/Questionnaire/index.js (hunk 3 above)
Comment (donia98) [reply client/src/Components/Pages/Questionnaire/index.js]: done

File: client/src/Components/Pages/Questionnaire/index.js (hunk 2 above)
Comment (donia98) [reply client/src/Components/Pages/Questionnaire/index.js]: Done

File: client/src/Components/Pages/Questionnaire/Wizard.js (hunk 1 above)
Comment (donia98) [reply client/src/Components/Pages/Questionnaire/Wizard.js]: what do you mean ?!

Comment (maxgerber) [reply client/src/Components/Pages/Questionnaire/Wizard.js]: it's a typo - you spell it "handle" not "hanle". never mind!

**Pull request merged**